composer.json

```diff
@@ -41,6 +42,10 @@
   "config": {
     "sort-packages": true
   },
+  "scripts": {
+    "cs-check": "phpcs --standard=vendor/spryker/code-sniffer/SprykerStrict/ruleset.xml -p src/",
```

Comment:
if you have tests dir that must also be mentioned here, and also run through in travis.

Reply:
added